src/scripts/main.js

```diff
@@ -1,7 +1,26 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const element = document.createElement('div');
+
+  element.className = 'notification';
+  element.classList.add(type);
+
+  element.style.top = `${posTop}px`;
+  element.style.right = `${posRight}px`;
+
+  element.insertAdjacentHTML(
+    'afterbegin',
+    `
+    <h2 class="title">${title}</h2>
+  <p>${description}</p>`,
+  );
+
+  document.body.appendChild(element);
+
+  setTimeout(() => {
+    element.remove();
+  }, 1000);
```

Comment:
The timeout for removing the notification is set to 1000 milliseconds (1 second), which might be too short for users to read the notification content. Consider increasing the timeout duration to give users more time to read the notification.